chapters/annotations.tex

```diff
@@ -1110,15 +1129,17 @@ \section{Annotations for the Graphical User Interface}\doublelabel{annotations-f
 \lstinline!0=f(u)!, and for every (potential,flow) pair of the form \lstinline!(v,i)!, providing
 an equation of the form \lstinline!0=f(v,i)!.
 
-{[}\emph{It is useful to check all models/blocks of a Modelica package
+\begin{nonnormative}
+It is useful to check all models/blocks of a Modelica package
 in a simple way. One check is to default connect every model/block and
-to check whether the resulting class is structurally consistent (= a
-stronger requirement as ``balanced''). It is rarely needed; but is for
+to check whether the resulting class is structurally consistent (which is a
+stronger requirement than being locally balanced). It is rarely needed; but is for
```

Comment:
Good replacement as->than, but I think we should keep it as "than being balanced".

Reply:
Done.